docs/API-functions.md

```diff
@@ -2057,9 +2051,92 @@ will provide this information on its own.
 
 The following flags are available, as collected in `logging_api.h` public header:
 
-* `SRT_LOGF_DISABLE_TIME`: Do not provide the time in the header
-* `SRT_LOGF_DISABLE_THREADNAME`: Do not provide the thread name in the header
-* `SRT_LOGF_DISABLE_SEVERITY`: Do not provide severity information in the header
-* `SRT_LOGF_DISABLE_EOL`: Do not add the end-of-line character to the log line
+- `SRT_LOGF_DISABLE_TIME`: Do not provide the time in the header
+- `SRT_LOGF_DISABLE_THREADNAME`: Do not provide the thread name in the header
+- `SRT_LOGF_DISABLE_SEVERITY`: Do not provide severity information in the header
+- `SRT_LOGF_DISABLE_EOL`: Do not add the end-of-line character to the log line
+
+## Time Access
+
+The following set of functions is intended to retrieve timestamps from clock used by SRT.
+This timestamp MUST be used by sender if it wants to explicitely assign a timestamp
+to a certain packet. The sender can pass the timestamp in `MSGCTRL::srctime` of `srt_sendmsg2(..)`
+function together with the packet being submitted to SRT.
+The time provided MUST equal or exceed connection start time (`srt_connection_time(..)`)
+of the SRT socket passed to `srt_sendmsg2(..)`.
+
+The current time value as of SRT internal clock can be retrieved using `srt_time_now()` function.
```

Comment:
```suggestion
The following set of functions is intended to retrieve timestamps from the clock used by SRT.
Such a timestamp MUST be used by the sender to explicitly assign a timestamp
to a certain packet. The sender can pass the timestamp in `MSGCTRL::srctime` of the `srt_sendmsg2(..)`
function together with the packet being submitted to SRT.
The time value provided MUST equal or exceed the connection start time (`srt_connection_time(..)`)
of the SRT socket passed to `srt_sendmsg2(..)`.

The current time value of the SRT internal clock can be retrieved using the `srt_time_now()` function.
```

Reply:
The second sentence: there's no obligation to use the timestamp (the default will be used otherwise), the conditional statement here is important. I don't know how to best express it.

What is important here is that if the sender procedure wants to explicitly specify the timestamp, it must be either the timestamp read from the SRT source when reading a packet (and it's provided in the `SRT_MSGCTRL::srctime` field when reading the packet), or a calculated time not earlier than the connection start time (obtained from `srt_connection_time()`).

Reply:
Maybe this way then:

The sender can pass the timestamp in `MSGCTRL::srctime` of the `srt_sendmsg2(..)`
function together with the packet being submitted to SRT.
If the `srctime` value is not provided (the default value of 0 is set), SRT will use internal
clock and assign the packet submission time as the packet timestamp.
If the sender wants to explicitly assign a timestamp
to a certain packet. this timestamp MUST be taken from SRT Time Access functions.
The time value provided MUST equal or exceed the connection start time (`srt_connection_time(..)`)
of the SRT socket passed to `srt_sendmsg2(..)`.

The current time value as of the SRT internal clock can be retrieved using the `srt_time_now()` function.